Android/app/src/main/java/com/etiennebaudoux/clipboardzanager/enums/DataEntryStatus.java

```diff
@@ -1,19 +1,26 @@
 package com.etiennebaudoux.clipboardzanager.enums;
 
-public enum DataEntryStatus
+import android.support.annotation.IntDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@IntDef({DataEntryStatus.ADDED, DataEntryStatus.DID_NOT_CHANGED, DataEntryStatus.DELETED})
+@Retention(RetentionPolicy.SOURCE)
+public @interface DataEntryStatus
 {
     /**
      * The data has been added locally and has not been synchronized yet.
      */
-    Added,
+    int ADDED = 1;
```

Comment:
Attention, be sure that those numbers corresponds to what we have here : [ClipboardZanager.Core.Desktop/Enums/DataStatus.cs](https://github.com/veler/clipboardzanager/blob/master/ClipboardZanager/Sources/ClipboardZanager.Core.Desktop/Enums/DataStatus.cs)

Reply:
The reason why is that the status is retrieves by the app on other device and is used to defines whether a data must be keep, deleted or else after a synchronization.